tcii18n/template.py

```diff
@@ -0,0 +1,15 @@
+from .translator import Translator
+
+_cached_translators = {}
+
+
+def flask_methods(app, get_translations_file):
+    @app.context_processor
+    def utility_processor():
+        def trans(sentence):
+            filename = get_translations_file()
+            if filename not in _cached_translators:
```

Comment:
I've already test it, and I confirm that this solution is working, I can also change with `if filename not in _cached_translators.keys():` if you prefer with way.